### PR TITLE
T1-19 Adding admin pages

### PR DESCRIPTION
**Pull request by jvera701**
screenshot

- Added admin table to see both users and foundations 
- Admin can click previous and next to navigate pages 
- Admin can click the check boxes to select which ones to delete, and then clicks the delete button 
- Added a pagination feature to the foundations page

### CODE REVIEW

File: src/components/FoundationsMenu.js [hunk 1]
```diff
@@ -1,42 +1,122 @@
...
+  setDisableNext(false);
+}
 
 const FoundationsMenu = () => {
-  const [foundations, setFoundations] = useState([])
+  const [foundations, setFoundations] = useState([]);
+  const [disablePrev, setDisablePrev] = useState(true);
+  const [disableNext, setDisableNext] = useState(false);
+  const [page, setPage] = useState(1);
+  const [route, setRoute] = useState(
```
Comment (leoneloliveros) [inline src/components/FoundationsMenu.js:49]: why are you using this hook? baseURL isn't define in axios config?

Comment (jvera701) [reply src/components/FoundationsMenu.js:49]: you mean the route variable ? I think you are right, I don't need to use that hook

Comment (leoneloliveros) [reply src/components/FoundationsMenu.js:49]: Yes, this project use axios configuration to declare baseURL one time in the project. The only thing you need to add in the path

Comment (leoneloliveros) [reply src/components/FoundationsMenu.js:49]: Please Fix this in all uploaded documents in this PR

**Pull request merged**